modules/uikit/includes/components/uiloader.h

```diff
@@ -3,8 +3,8 @@
 
 #include "widget.h"
 
-#include "stylesheet.h"
-#include "uidocument.h"
+#include "resources/stylesheet.h"
+#include "resources/uidocument.h"
```

Comment:
this will not work with SDK directory structure

Reply:
Can return that back to the way it was. Do the other uikit source files that use "resources/stylesheet.h" need to be changed to conform or is this just that specific file?

Reply:
Went ahead and updated both `modules/uikit/includes/components/label.h` and `modules/uikit/includes/components/widget.h` since they were the main ones that didn't fit the structure.